app/2.0/docs/devguide/encapsulated-style.md

```diff
@@ -0,0 +1,639 @@
+---
+title: Encapsulated style with Polymer 2.0 and shadow DOM v1
+---
+
+<!-- toc -->
+
+## Styling your elements
+
+Polymer supports DOM templating and the shadow DOM API. When you use the `<dom-module>` and `<template>` blocks to template your custom element, Polymer attaches a shadow root to instances of this element. Polymer then copies in the contents of the template you provided for your element.
+	
+Here's an example:
+
+[See it on Plunker](http://plnkr.co/edit/VPktVlI0PWMnYdYVsVBx?p=preview)
+
+```html
+<!-- Create a template for the custom element -->
+<dom-module id='custom-element'>
+  <template>
+    <h1>Heading!</h1>
+    <p>We are elements in custom-element's local DOM.</p>
+  </template>
+</dom-module>
+
+<!-- Drop the custom element on the page -->
```

Comment:
The import link imports the polymer library. The polyfills are loaded using a `<script>` tag. In 1.x this was always `webcomponentsjs/webcomponents-lite.js` ... (Now we have a new option, `webcomponentsjs/webcomponents-loader.js`, which automatically loads the correct set of polyfills for the current browser.)

http://plnkr.co/edit/dnL9SeRFZr0sQt0pqKaG?p=preview

The polyfills can only be imported _once_, so you typically load them in the main document. 

HTML imports get deduplicated by the browser, so every custom element can have a `<link rel="import">` at the top of it for Polymer, but Polymer only gets loaded once. Each element should import its own dependencies this way.